cscs-checks/microbenchmarks/stream/stream.py

```diff
@@ -24,8 +24,10 @@ def __init__(self):
                                          'PrgEnv-gnu-nompi',
                                          'PrgEnv-pgi-nompi']
 
+        self.use_multithreading = False
+
         self.prgenv_flags = {
-            'PrgEnv-cray': ['-homp'],
+            'PrgEnv-cray': ['-fopenmp'],
```

Comment:
Why not also here `-O3`. It doesn't make a big difference in this case, but it's better if you put it.

Reply:
I think we have the `-O3` set in a different place in the test. I believe it's better to set all the flags here.